### DISCUSSION TITLE
Publish AvaloniaResource ,but does not exist in publish folder.

### DISCUSSION THREAD

**Discussion by SinnoSong** [AvaloniaUI/Avalonia#9008] [Q&A] [answered]
I use this ```dotnet publish lock_screen_linux.csproj  -c Debug -r linux-x64 --self-contained```  command to publish application.But all AvaloniaResource does not exist in the publish folder.Here is the Screenshot。
![image](https://user-images.githubusercontent.com/59649274/191933798-379cdb63-830a-40ba-a6ab-3dee6129c24e.png)
![image](https://user-images.githubusercontent.com/59649274/191933887-5d05d150-7f13-4a2e-9cb3-dec077a0c2fa.png)
![image](https://user-images.githubusercontent.com/59649274/191933979-44ccd151-abbb-481e-b7ed-0904f5c93389.png)

**Comment by timunie** [comment, marked as answer]
AvaloniaResource is included inside the dll itself. If you need the file in output dir, set copy to output in your ide on the folder or file. 

https://social.technet.microsoft.com/wiki/contents/articles/53248.visual-studio-copying-files-to-debug-or-release-folder.aspx#:~:text=Copy%20to%20Output%20Directory%20is%20a%20property%20for,Output%20Directory%20is%20set%20to%20%5BDo%20not%20copy%5D.

**Reply by SinnoSong** [reply]
Thank you very much!
